CHANGELOG.md

```diff
@@ -16,7 +16,8 @@
     -   Changed the low camera angle limit to 32 degrees from 10 degrees.
     -   `onCombineExit` action will now fire alongside the `onCombine` action.
     -   Newly created contexts will no longer be autoselected.
-    -   Toast messages will now only remain on screen for 2 seconds.
+    -   Toast messages will now only remain on screen for 2 seconds by default.
+    -   Added ability to set duration of toast, `plater.toast("message", durationNum)`.
```

Comment:
@shaneYeti Can you add this note to the latest release section? It should be in `0.10.4` I think